.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,66 @@
+---
+name: Bug Report
+description: Create a report to help us improve
+title: "[Bug]: "
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Also tell us, what did you expect to happen?
+      placeholder: Tell us what you see!
+    validations:
+      required: true
+  - type: input
+    id: version
+    attributes:
+      label: Version
+      description: What version of our software are you running?
```

Comment:
I think it may be good to capture the Xcode version somewhere

Reply:
Swift version too? Not sure

Reply:
Done